docs/getting-started/supported-languages-and-tools.md

```diff
@@ -200,7 +200,8 @@ The table below lists all languages and frameworks that Codacy supports and the
     </tr>
     <tr>
       <td>Objective-C</td>
-      <td><a href="http://fauxpasapp.com/">Faux Pas</a><a href="#client-side"><sup>1</sup></a></td>
+      <td><a href="http://fauxpasapp.com/">Faux Pas</a><a href="#client-side"><sup>1</sup>,
+          <a href="https://clang.llvm.org/extra/clang-tidy/">Clang-Tidy</a><a href="#client-side"><sup>1</sup></a></td>
```

Comment:
Just a very minor nitpick, we're currently listing the tools in alphabetical order - in a way, this tries to convey that no tool for each language is preferred against the others. So I would include Clang-Tidy first:

```suggestion
      <td><a href="https://clang.llvm.org/extra/clang-tidy/">Clang-Tidy</a><a href="#client-side"><sup>1</sup>,
          <a href="http://fauxpasapp.com/">Faux Pas</a><a href="#client-side"><sup>1</sup>
          </a></td>
```

Reply:
I amended the commit with this change ✅